terraform/modules/rabbitmq/helm/main.tf

```diff
@@ -0,0 +1,15 @@
+resource "helm_release" "custom_helm_release" {
```

Comment:
This is duplicated helm module code. I had a same problem with `helm` module source not being found, I worked around it based on this [issue](https://github.com/hashicorp/terraform/issues/23333) by creating symlink like this `ln -s ../helm helm`, this way we don't need to duplicate module code.